actions/guest_script_run.yaml

```diff
@@ -0,0 +1,49 @@
+---
+  name: guest_script_run
+  runner_type: orquesta
```

Comment:
If we add an Orquesta workflow here, we'll also need to update `pack.yaml` to include `stackstorm_version: ">=2.9.0"`

Reply:
I changed it to an Orquesta workflow because of https://github.com/StackStorm/st2/issues/4368.

Reply:
It's totally fine using Orquesta, but the issue is Orquesta is not available in ST2 < 2.9. So if we're going to add Orquesta workflows to this pack, we need to update the metadata to require a minimum ST2 version.

Reply:
The change to `packs.yaml` was made.